components/support/ktx/src/main/java/mozilla/components/support/ktx/android/content/Context.kt

```diff
@@ -24,6 +25,8 @@ inline fun <reified T> Context.systemService(name: String): T {
     return getSystemService(name) as T
 }
 
+fun Context.getAccessibilityManager() = getSystemService(Context.ACCESSIBILITY_SERVICE) as? AccessibilityManager
```

Comment:
Let's add some KDoc?

Also this could either be private or move inside isScreenReaderEnabled(). It looks like we do not want to expose this publicly?

Reply:
Yes. It should be moved inside isScreenReaderEnabled()